apps/main/src/llamalend/PageLlamaMarkets/cells/RateCell/RewardsTooltipItems.tsx

```diff
@@ -29,7 +30,10 @@ export const RewardsTooltipItems = ({
       {hasIncentives && <TooltipItem title={title}>{percentage}</TooltipItem>}
       {extraIncentives?.map(({ percentage, title }, i) => (
         <TooltipItem key={i} subitem title={title}>
-          {formatPercent(percentage)}
+          <Stack direction="row">
+            <RewardsImg src={RCCrvLogoMD} alt={t`CRV`} width={20} />
```

Comment:
I had to double check if these were really only CRV rewards, and not possibly some other token.
So the only place where these `extraIncentives` are being used, is in `LendRateTooltip.tsx`
![image](https://github.com/user-attachments/assets/29138deb-76b4-48f6-bb9f-3468c30f5f6b)
Here, we're taking just `lendCrvAprUnboosted` and putting it in an array of size 1.

Imo we need to make a choice here:

a) Either we get rid of the `extraIncentives` array property and replace it with a single more descriptive number, like for example `extraCrvIncentive`.
b) Or we don't assume that the rewards are CRV rewards and keep it as it is, allowing other tokens to be rewards in the future.

Reply:
We will have more rewards in the future when the information is available regarding yield bearing tokens.
So, I'll just make sure to use the same hook here.